src/editor/screens/AddActivityCard.js

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {
+  SafeAreaView,
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  useState
+} from 'react-native';
+import TagFilter from '../components/TagFilter';
+import BackArrow from '../../../assets/backArrow.svg';
+import { Searchbar } from 'react-native-paper';
+import { mdiMagnify } from '@mdi/js';
+import { isSearchBarAvailableForCurrentPlatform } from 'react-native-screens';
+
+const TAGS = [
+  'Warm Up',
+  'No Equipment',
+  'Beginner',
+  'Rhythm',
+  'Note Reading',
+  'Group Activity',
+  'Quick',
+  'Scale'
+];
+
+const AddActivityCard = function () {
+  const [searchQuery, setSearchQuery] = React.useState('');
+
+  const onChangeSearch = query => setSearchQuery(query);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={{ paddingHorizontal: '3%' }}>
+        <TouchableOpacity style={styles.backButton}>
+          <BackArrow height={15} width={15} />
+          <Text style={styles.backText}> Back </Text>
+        </TouchableOpacity>
+        <Text style={styles.header}> Main Lesson Activity Card </Text>
+        <Text style={styles.tags}> Tags: </Text>
+        <View style={styles.tagContainer}>
+          {TAGS.map((tag, index) => (
+            <TagFilter key={index} tagContent={tag} />
+          ))}
+        </View>
+        <Searchbar
+          placeholder="Search by title or keyword"
+          onChangeText={onChangeSearch}
+          value={searchQuery}
+          icon={require('../../../assets/search.png')}
+          style={{ backgroundColor: 'white', marginTop: '3%' }}
+        />
+      </View>
+    </SafeAreaView>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+    width: '100%'
+  },
+  tagContainer: {
+    flexDirection: 'row',
+    flexWrap: 'wrap'
+  },
+  header: {
+    color: 'black',
+    fontSize: 22,
+    fontFamily: 'Poppins-ExtraBold',
```

Comment:
Not 100% confident, but I think that most of the designs only use Poppins or Mulish. I f you want it to be bolder consider changing font weight (I haven't looked at designs for this particular instance so ignore if I'm wrong)

Reply:
I'm so glad fonts are working now :D

Reply:
I think Baker is right. From last standup I think we mentioned we only use Poppins!